package/kernel/linux/modules/leds.mk

```diff
@@ -145,3 +145,19 @@ define KernelPackage/leds-pca963x/description
 endef
```

Comment:
this shuould be added to target/linux/x86/modules.mk

Reply:
Ok will do this